src/core/ContactsManager.cpp

```diff
@@ -186,3 +186,13 @@ void ContactsManager::onUnreadCountChanged()
     emit unreadCountChanged(user, model->unreadCount());
 }
```

Comment:
I mean, I'm all for "encapsulation" as a reason, but in my opinion this would be a lot more straightforward to follow.